tests/integration/test_password_rotation.py

```diff
@@ -122,6 +159,16 @@ async def test_empty_password(ops_test: OpsTest) -> None:
     assert password == "None"
 
 
+@pytest.mark.group(1)
+async def test_db_connection_with_empty_password(ops_test: OpsTest):
+    """Test that user can't connect with empty password."""
+    primary = await get_primary(ops_test, f"{APP_NAME}/0")
+    address = get_unit_address(ops_test, primary)
+    with pytest.raises(psycopg2.Error):
+        with db_connect(address, "") as connection:
```

Comment:
This will test user `operator` only. Should we test lack of connection without password using user `postgresql` and the rest from the list https://charmhub.io/postgresql/docs/e-users
@marceloneppel what do you think, is it necessary?

Reply:
I think it's not necessary to test with the other users.